lib/loadGltfUris.js

```diff
@@ -169,7 +168,7 @@ var jimpSupportedExtensions = ['.png', '.jpg', '.jpeg', '.bmp'];
 function generateJimpImage(object) {
     var pipelineExtras = object.extras._pipeline;
     var ext = pipelineExtras.extension;
-    if (jimpSupportedExtensions.indexOf(ext) === -1) {
+    if (supportedExtensions.test(ext)) {
```

Comment:
Don't be so hard on yourself.  We are were all first time committers at some point.  We are happy to help and appreciate your help.